WIP-CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - The sc-logs-retention cronjob now runs without error even if no backups were found for automatic removal
 - Harbor Swift authentication configuration options has moved from `citycloud` to `harbor.persistence.swift`.
 - The dry-run and apply command now have the options to check against the state of the cluster while ran by using the flags "--sync" and "--kubectl".
+- The dex chart is upgraded from stable/dex 2.15.2 to dex/dex 0.3.3.
```

Comment:
This is nice, but I was thinking of the application version, not the chart version (but you can keep info about chart version as well)